### PR TITLE
Docs#1761

### PR DESCRIPTION
**Pull request by dvagnero**
- add procedure for Cache CR update strategies 
- add info that HR connections through Route require encryption
- fix table
- make cache templates and logging configuration visible in the table of content

### CODE REVIEW

File: documentation/asciidoc/topics/con_hotrod_clients.adoc [hunk 1]
```diff
@@ -13,7 +14,14 @@ Clients running in the same {k8s} cluster can access internal IP addresses for {
...
 `HASH_DISTRIBUTION_AWARE` is the default intelligence mechanism and enables clients to route requests to primary owners, which provides the best performance for Hot Rod clients.
 
-.`BASIC` intelligence
+Clients running on a different {k8s}, or outside {k8s}, can access {brandname} through a `LoadBalancer`, `NodePort`, or {openshiftshort} `Route`.
+
+[IMPORTANT]
+====
+Hot Rod clients connections via {k8s} `Route` require encryption.
+You must configure TLS with SNI otherwise the Hot Rod connection fails. +
+For unencrypted Hot Rod client connections, you must use a `LoadBalancer` service.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/con_hotrod_clients.adoc:23]: A NodePort service is also supported.

**Comment by dvagnero** [comment]
Do you want me to squash the commits?

**Comment by ryanemerson** [comment]
> Do you want me to squash the commits?

Yes please. Feel free to keep individual logs in the commit that describe the changes , but commits with messages like "improvements based on peer review" don't make a lot of sense in the branch git logs.

**Comment by dvagnero** [comment]
I hope this works. And I agree, I want to get little bit more organized and have meaningful commit messages :)

**Comment by ryanemerson** [comment]
Thanks @dvagnero

**Pull request merged**